docs/proposals/20191017-kubeadm-based-control-plane.md

```diff
@@ -148,6 +151,7 @@ Non-Goals listed in this document are intended to scope bound the current v1alph
 8. As a cluster operator, I want to be able to quickly respond to a non-Kubernetes CVE that affects my base image or Kubernetes dependencies by upgrading my clusters in an automated fashion.
 9. As a cluster operator, I would like to upgrade to a new minor version of Kubernetes so that my cluster remains supported.
 10. As a cluster operator, I want to know that my cluster isn’t working properly after creation. I have ended up with an API server I can access, but kube-proxy isn’t functional or new machines are not registering themselves with the control plane.
+11. As a cluster operator I would like to upgrade my control plane without any extra resources.
```

Comment:
For sake of completeness, IMO we should add a used case for scale out as well.
Also, it would be nice to talk about upgrade strategies, so we use wording consistent with the goals.

Reply:
New use case added.